### DISCUSSION TITLE
Epub

### DISCUSSION THREAD

**Discussion by zeoint** [mienaiyami/yomikiru#218] [General]
The custom styles applied to epub's via yomikiru epub reader settings like paragraph spacing, font style, size, etc.
Can they still be applied to epub's if there is no stylesheet for the epub?

**Comment by mienaiyami** [comment]
Yes, given it has right html.

**Reply by zeoint** [reply]
I have removed stylesheet references from epub file and extracted these two .xhtml files.
Will customizing of these work without stylesheet? Can you please check?

[Test.zip](https://github.com/mienaiyami/yomikiru/files/12206611/Test.zip)

**Reply by mienaiyami** [reply]
It will.
![image](https://github.com/mienaiyami/yomikiru/assets/84740082/7eee9d6d-ee9c-4970-8acf-6c133e031cd1)

Line height and word spacing is applied to text as whole, paragraph spacing is only applied when there are <p> in the html.

**Reply by zeoint** [reply]
> paragraph spacing is only applied when there are
in the html.

When there are 'p tags' in the html, I guess.

Thanks for the clarification.
